### ISSUE TITLE
Installation instructions of NiaPy-examples/CEC

### ISSUE THREAD

**Issue by rhododendrom** [NiaOrg/NiaPy-examples#5]
- all program parameters should have a clear explanation [Set the benchmark competition to use], options [options: `8, 13, 14, 15, 17, 18`] and an example [e.g. '-c 18']. Some program parameters lack an example.
- run command: program parameter '-a' missing description.
- run command: program parameter '-r' missing description.
- run command: program parameter '-D' missing description: is it dimension? Then '-d' should be used.
- write generation should be explained a bit more in detail. Does this generate an external file? Where is its root directory and how is it named? Does a new run overrun previous one?
- link should be added to list of benchmark functions, otherwise is hard to follow the meaning of '-f 12'.
- copy README content from folder CEC to CECC (with adjustments).

**Comment by GregaVrbancic** [comment]
@kb2623 can you look at this?
